root_main.tf

```diff
@@ -24,7 +25,12 @@ module "ses-eu-west-1" {
   project               = var.project
   environment_full_name = lookup(local.environment_full_name_map, local.environment)
   hosted_zone_id        = module.route_53_zone.hosted_zone_id
+  dns_delegated         = local.environment == "prod" ? false : true
   providers = {
```

Comment:
What are these variables used for? I can see them defined in the modules variables file but they then aren't used anywhere else I can see.

Reply:
tdr-terraform-modules updated to use these variables in conditionals